docs/FAQs.md

```diff
@@ -21,11 +21,16 @@ Provisioners are heterogeneous, which means that the nodes they manage are sprea
 ### What if my pod is schedulable for multiple Provisioners?
 It's possible that an unconstrained pods could flexibly schedule in multiple groups. In this case, Provisioners will race to create a scheduling lease for the pod before launching new nodes, which avoids unnecessary scale out.
 ## Reallocation
-TODO
+### How does Karpenter terminate nodes?
+Karpenter annotates nodes that are underutilized with a TTL. If the node remains underutilized after the TTL expires, Karpenter then cordons the node and uses the Kubernetes Eviction API to evict all non-daemonset pods. Once the node is empty, the node is terminated.
```

Comment:
can you add links for `cordons` and `Kubernetes Eviction API`. And can you be explicit about what `terminated` means exactly ("using the cloud provider" or something?)